engine/lite/k8sDeploymentManager/src/test/scala/pl/touk/nussknacker/k8s/manager/K8sDeploymentManagerUnitTest.scala

```diff
@@ -0,0 +1,51 @@
+package pl.touk.nussknacker.k8s.manager
+
+import org.scalatest.{FunSuite, Matchers}
+import pl.touk.nussknacker.engine.api.ProcessVersion
+import pl.touk.nussknacker.engine.api.process.{ProcessId, ProcessName, VersionId}
+import pl.touk.nussknacker.k8s.manager.K8sDeploymentManager.{labelsForScenario, objectNameForScenario, scenarioIdLabel, scenarioNameLabel, scenarioVersionLabel}
+import org.scalatest.prop.TableDrivenPropertyChecks._
+
+class K8sDeploymentManagerUnitTest extends FunSuite with Matchers {
+
+  private def versionForName(name: String) = ProcessVersion(VersionId(13), ProcessName(name), ProcessId(256), "user", None)
+
+  test("should generate correct labels for scenario names") {
+
+    val names = Table(
+      ("scenario name", "name label"),
+      ("standard", "standard-fe6d3468cf"),
+      ("ała", "a-a-b7ec948d4b"),
+      //same as above after sanitize, but hash is different
+      ("a-a", "a-a-199d9b3dfc")
+    )
+    forAll(names) { (scenarioName: String, nameLabel: String) =>
+      labelsForScenario(versionForName(scenarioName)) shouldBe Map(
+        scenarioNameLabel -> nameLabel,
+        scenarioVersionLabel -> "13",
+        scenarioIdLabel -> "256"
+      )
+    }
+  }
+
+  test("should generate correct object id for scenario names") {
+
+    val names = Table(
+      ("scenario name", "hashInput", "object id"),
+      ("standard", None, "scenario-256-standard"),
+      ("ała", None, "scenario-256-a-a"),
+      //here we don't care about hash, as id should be unique
+      ("a-a", None, "scenario-256-a-a"),
+      //but for different content, id should be different
+      ("st", Some("version1"), "scenario-256-st-fc6af3ec64"),
+      ("st", Some("version2"), "scenario-256-st-366a963b60"),
```

Comment:
Can u add two test cases wit scenario name overflowing limit of 63 characters but different hashes? it was the case that was most important for me

Reply:
thanks